setup.py

```diff
@@ -104,5 +113,5 @@ def get_version(version_file):
         ]
     },
     # Conveniently allows one to run the CLI tool as `example`
```

Comment:
You missed some skeleton text here:
```suggestion
    # Conveniently allows one to run the CLI tool as `pe-reports`
```

Reply:
Resolved in da8dcea